examples/freertos/explorer_board/src/gpio_ctrl/gpio_ctrl.c

```diff
@@ -32,26 +32,24 @@ static void button_callback(rtos_gpio_t *ctx, void *app_data, rtos_gpio_port_id_
 
 static void volume_up( void )
 {
-    BaseType_t gain = 0;
-    gain = audiopipeline_get_stage1_gain();
-    if( gain != 0xFFFFFFFF )
+    BaseType_t gain = audiopipeline_get_stage1_gain();
+    if( gain < 60 )
```

Comment:
Could you add an appconfAUDIOPIPELINE_MAXIMUM_GAIN to appconf.h to replace this magic number.